src/integration/utils/mergeTwoObjectsIntoOne.ts

```diff
@@ -0,0 +1,13 @@
+export const mergeTwoObjectsIntoOne = (
```

Comment:
What about:
```ts
return {...majorObject, ...secondaryObject}
```

Reply:
I will keep the function, but change it to use spread operator.
because merging order is important, it should be  {...secondaryObject,...majorObject}